DeAceroDLabs/marketplace-frontend
# [DMP-449] Creación de usuario

## What is done

[[DMP-449] Creación de usuarios en Deacero y Auth0](https://deacero.atlassian.net/browse/DMP-449)

## How is done

1. Created a new api file for handling user-related calls..
2. Created a new method for handling user creation on Auth0.

## How to test

1. Intall new dependencies `npm i`
2. Run `npm start`.
3. Go to `/signup`.
4. Fill the form and submit it.
5. You should see the success popup if the account was created successfully.
6. Go to [mkt-place app dashboard](https://manage.auth0.com/dashboard/us/mkt-place/) using the credentials shared via slack.
7. Go to `Try your Login box``
8. Use the credentials you used for creating your user. You should be redirected to a success page.
